src/pages/index.js

```diff
@@ -2,101 +2,246 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Needs correction: when I open the profile form the text in the description input is always equal to `undefined`